crates/binjs_meta/src/spec.rs

```diff
@@ -106,6 +106,7 @@ pub struct Field {
     name: FieldName,
     type_: Type,
     documentation: Option<String>,
+    is_lazy: bool,
```

Comment:
I believe it would be less error-prone to have an enum `Laziness` instead of a boolean.

I put something along these lines [in that patch](https://github.com/binast/binjs-ref/commit/8a875327360e847276fe793332bf21d6c73b56ec).